### ISSUE TITLE
enhancement: EMC "shredder" or one-way EMC burn block.

### ISSUE THREAD

**Issue by preceptor-teeth** [pkmnfrk/equivalentintegrations#21]
Saw this idea posted on the projectE repository, thought it would be a good fit here:

https://github.com/sinkillerj/ProjectE/issues/1767

**Comment by pkmnfrk** [comment]
So, let me rephrase this a different way, just to make sure I understand:

* Add a block that functions similarly to the Transmutation Chamber
* This block is one way, only able to burn items, not extract them
* Items are still learned, they just can't be materialized
* It should be cheaper than the Transmutation Chamber

I'm not sure that "trash can" is the most appropriate name, that sounds like it destroys EMC, which confused me for a moment. Maybe a "Transmutation Receptor" or "Transmutation Shredder"

**Comment by preceptor-teeth** [comment]
> So, let me rephrase this a different way, just to make sure I understand:
> 
> * Add a block that functions similarly to the Transmutation Chamber
> * This block is one way, only able to burn items, not extract them
> * Items are still learned, they just can't be materialized
> * It should be cheaper than the Transmutation Chamber
> 
> I'm not sure that "trash can" is the most appropriate name, that sounds like it destroys EMC, which confused me for a moment. Maybe a "Transmutation Receptor" or "Transmutation Shredder"

This wasn't my original idea, just saw it and thought it made a good fit here.  With that being said, I think you got the gist!  I think on packs that gate ProjectE very late this would be very useful.  It may even have a setting to block non-emc items, or destroy non-emc items so it could be used as a "trash can" that dosen't waste EMC where applicable.

I like "Transmutation Shredder"  Sounds good.  :)

**Comment by Sandstroem** [comment]
To clarify. The name trashcan came from the comparison to the Exu trashcan. It should trash/burn items and convert them to emc, stored in the player's emc data.

You remember all those kitchens you got on PO2? That would be an example of stuff cluttering your storage. That would be something I dump into the new block.

Whether it learns the items or not is open for discussion. Not learning would be OK and I might even lean towards that, but it could also be a config option.

**Comment by pkmnfrk** [comment]
Sorry for the delay on this, but here's what I'm going to do. The Transmutation Disassembler is going to basically be a trashcan that makes EMC as suggested above. It will function identically to a Transmutation Chamber, with the following exceptions:

1. It will be significantly cheaper. I'm thinking that it'll be the same recipe, but with a Transmutation Table instead of a Transmutation Tablet (but, this is yet to be finalized)
2. It will not feature the efficiency mechanic, instead always having max efficiency. (This is separate from ProjectE's transmutation cost mechanic)

Notably, it means that if you want it to learn items, you give it an Alchemical Algorithms item, just like the Chamber.

Visually, it looks like a Chamber, but more like a trash can. I guess I can just post a screenshot, huh:

![image](https://user-images.githubusercontent.com/1333816/53057050-99f36e00-347b-11e9-9ed5-d03b5021bb16.png)

**Comment by pkmnfrk** [comment]
Oops, there is one additional difference:
3. A Transmutation Chamber that is unable to learn will reject items if the owner doesn't know them. A Transmutation Disassembler will always consume items and produce EMC, but if it cannot learn, then you won't be able to reproduce them later (unless you learn them via another method).

**Comment by pkmnfrk** [comment]
This will be included in 0.3.0
